interfaces/HO-Portal/src/app/models/person.model.ts

```diff
@@ -14,17 +14,29 @@ export class Person {
   selectedForValidationDate?: string;
   validationDate?: string;
   inclusionDate?: string;
+  status: PaStatus;
 }
 
 // Model for display (in table)
 export class PersonRow {
   did: string;
   checkboxVisible: boolean;
   pa: string; // Display label
+  status: PaStatus;
```

Comment:
This one should not be of type `PaStatus`, as it would containt the translated string-version..